emcc.py

```diff
@@ -147,7 +147,10 @@
 # These symbol names are allowed in INCOMING_MODULE_JS_API but are not part of the
 # default set.
 EXTRA_INCOMING_JS_API = [
-  'fetchSettings'
+  'fetchSettings',
+  'preloadedImages',
+  'preloadedAudios',
+  'preloadedWasm',
```

Comment:
Do we even need this? I can't actually remember a use case for it.

Reply:
I mean, why would code set `Module.preloadedImages` on startup?

Reply:
I wasn't sure if that was valid.. but its looks like is at best unlikely.. and maybe completely invalid.  So I can just remove.

I agree these looks like they date back to a time when the file_packager output would use them..

Reply:
Maybe it's safer to leave it for now. As it is not in the main list of INCOMING_JS it has no cost basically.